recipes/descwl-shear-sims/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "descwl-shear-sims" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
```

Comment:
We've been taking the convention that DESC software on conda-forge should be prefixed with `lsstdesc-`. This helps with discoverability, namespace issues, etc. 

What do you all think we should do here?

Reply:
I think `lsstdesc-descwl-shear-sims` is redundant and awkward

Reply:
I think its OK.  The main confusion is it doesn't match the package name.

Reply:
What about `lsstdesc-wl-shear-sims`?